go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -323,6 +322,12 @@ func setupOVNNode(node *kapi.Node) error {
 		)
 	}
 
+	// In the case of DPU, the hostname should be that of the DPU and not the K8s Node.
+	// So, skip setting the incorrect hostname.
+	if config.OvnKubeNode.Mode != types.NodeModeDPU {
```

Comment:
I believe this on it's own will break the DPU functionality. We need the hostname to be set by the DPU. Could we move this out of this PR?